app/models/content_extraction/parsers/email_alert_signup_parser.rb

```diff
@@ -0,0 +1,11 @@
+class ContentExtraction::Parsers::EmailAlertSignupParser
```

Comment:
I would remove the suffix `Parser` as we are already in a `Parsers` module. This apply to all parsers.

Reply:
Addressed in https://github.com/alphagov/content-performance-manager/pull/576